build/params_2k.go

```diff
@@ -67,9 +67,9 @@ var UpgradeThunderHeight = abi.ChainEpoch(-23)
 
 var UpgradeWatermelonHeight = abi.ChainEpoch(-24)
 
-var UpgradePineappleHeight = abi.ChainEpoch(20)
+var UpgradeDragonHeight = abi.ChainEpoch(20)
 
-var UpgradeMangoHeight = UpgradePineappleHeight + 10
+var UpgradeMangoHeight = UpgradeDragonHeight + 10
```

Comment:
Per discussion, we'll probably wind up going with something like `+ 100` instead. But that doesn't have to change in this PR.

Reply:
Do we also want the Drand switch to be `+100` epochs away from the nv22 upgrade in the butterfly/calibration/mainnet ?

Reply:
> Do we also want the Drand switch to be +100 epochs away from the nv22 upgrade in the butterfly/calibration/mainnet ?

We really _only_ care about mainnet.

@arajasek can we make this 120 so it falls on an hour mark?